core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -969,6 +973,86 @@ class PlaintextConsumerTest extends BaseConsumerTest {
     }
   }
 
+  @ParameterizedTest
+  @ValueSource(strings = Array(CooperativeStickyAssignor.COOPERATIVE_STICKY_ASSIGNOR_NAME, RangeAssignor.RANGE_ASSIGNOR_NAME))
```

Comment:
It needs a full class name, to be passed into PARTITION_ASSIGNMENT_STRATEGY_CONFIG config, ex:
`@ValueSource(strings = Array(classOf[CooperativeStickyAssignor].getName, classOf[RangeAssignor].getName))`

Reply:
i cannot use classOf in Annotation, it will get "annotation argument needs to be a constant" error. Just set the full class name instead